form/src/main/java/com/ritense/form/web/rest/FormManagementResource.java

```diff
@@ -44,6 +44,9 @@ public interface FormManagementResource {
     @GetMapping(value = "/v1/form-management/{formDefinitionId}", consumes = MediaType.ALL_VALUE)
     ResponseEntity<? extends FormDefinition> getFormDefinitionById(String formDefinitionId);
 
+    @GetMapping(value = "/v1/form-management/exists/{name}", consumes = MediaType.ALL_VALUE)
+    ResponseEntity<? extends Boolean> getExistsByName(String name);
```

Comment:
How is this implemented on the angular side?

Reply:
In file https://github.com/valtimo-platform/valtimo-frontend-libraries/pull/348/files#diff-50418381f5441e7ddf2cdd68c3de00928c76a5b8d330e49442b251badc06d864
It's a validator that checks the call to this backend, this validator has been added to the duplicate-form screen as well as the new-form screen.